### PR TITLE
Phabricator: fetch data on changed files in revisions

### PR DESCRIPTION
**Pull request by cjwooo**
## Description

> Provide description here

## Type of change
- [ ] Bug fix
- [x] New feature
- [ ] Breaking change

## Related issues

> Fix [#1]() 

## Migration notes

> Describe migration notes if any

## Extra info

> Add any additional information

### CODE REVIEW

**Comment by cjwooo** [comment]
There's still a large diff on package-lock.json. Even though the latest lerna version now follows the file's existing spacing convention, lerna and bare npm still organize the json structure differently :( I haven't found a way to reconcile the two yet.

**Review by tovbinm — APPROVED** [review]

**Comment by tovbinm** [comment]
is it possible to add a test?

File: sources/phabricator-source/src/phabricator.ts [hunk 1]
```diff
@@ -378,28 +388,82 @@ export class Phabricator {
...
+          revision: {
+            id: revision.id,
+            phid: revision.phid,
+            dateModified: revision.fields?.dateModified,
+          },
+          repository: revision.repository,
+          filesChanged: uniq(
+            files.flatMap((f) => [f.from, f.to]).filter((f) => f)
+          ),
+          linesAdded: files.reduce((total, file) => total + file.additions, 0),
```
[the remark this answers is not in the document]
Comment (cjwooo) [reply sources/phabricator-source/src/phabricator.ts]: No, `parseDiff` always returns an array.

**Comment by cjwooo** [comment]
Forgot to push the converter before making the PR.

**Pull request merged**